content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Better Together: Building Efficient Microservices in Kubernetes using WebAssembly"
+image: "images/blogs/adobe-kubernetes/header.png"
+date: 2022-11-17T11:00:00-05:00
+author: "Sean Isom and Colin Murphy, Adobe"
+description: "Bringing two major CNCF projects together – WebAssembly and Kubernetes – promises greater agility and major efficiencies"
```

Comment:
this implies that webAssembly is a CNCF project... but the CNCF project here is wasmCloud

Reply:
Fixed 👍🏻